src/blots/Link.js

```diff
@@ -1,7 +1,5 @@
 import Inline from 'quill/blots/inline';
 
-import './Link.scss';
```

Comment:
We should style links in applications which will use post-editor

Reply:
please add a comment on that